components/Blog/index.js

```diff
@@ -0,0 +1,5 @@
+import BlogPage from './blogPage'
+import '../../node_modules/tachyons-sass/tachyons.scss'
```

Comment:
There should not be a need to import tachyons.scss like this, as it is imported into our main.scss file. Additionally, I don't see the point of this entire file, as it's just importing and exporting. Import the scss files on the actual components that are to be styled.

Reply:
Thanks for letting me know... I removed `import '../../node_modules/tachyons-sass/tachyons.scss'` from this file... I choose `'./blog.scss'`

Reply:
Please refer to `pages/blog.js` file where I'm importing from `../components/Blog` not `../components/Blog/index`... Additionally, `index` file acts as an encapsulation for that particular module as other modules don't need to know about the exact files in that particular module where I'm exporting the complete component... In this case `component/blog/blogPage` acts as entry files... If other modules wants to use my blog component they need to refer to this particular module `component/blog/blogPage` which is a hassle for large number of components